scripts/lib/CIME/test_scheduler.py

```diff
@@ -638,6 +636,7 @@ def _xml_phase(self, test):
                         envtest.set_test_parameter("NTASKS_"+comp, "1")
                         envtest.set_test_parameter("NTHRDS_"+comp, "1")
                         envtest.set_test_parameter("ROOTPE_"+comp, "0")
+                        envtest.set_test_parameter("PIO_TYPENAME", "netcdf")
```

Comment:
Why would you hard code this?  You can set it in config_pio.xml if you need to.

Reply:
@amametjanov , can you comment here?

Reply:
Tasks, threads and rootpe are already hard-wired. Setting PIO_TYPENAME=netcdf here keeps all SCM-specific settings nearby. Error message is at https://github.com/E3SM-Project/E3SM/pull/3300#issuecomment-551270871